### ISSUE TITLE
Instagram link unreadable and nothing is saved

### ISSUE THREAD

**Issue by jgroeneveld** [DominikPieper/obsidian-ReadItLater#164]
### Please check that this issue hasn't been reported before.

- [X] I searched previous [Bug Reports](https://github.com/DominikPieper/obsidian-ReadItLater/labels/type%3A%20bug) didn't find any similar reports.

### Expected Behavior

The link is saved as usual, at least as a plain link.

### Current behaviour

I see the message 
_@mozilla/readability considers this document to unlikely be readerable._
and nothing is saved

### Steps to reproduce

Paste https://www.instagram.com/reel/C-w3D8BMvsh/ either on mac or iphone

### Which Operating Systems are you using?

- [ ] Android
- [X] iPhone/iPad
- [ ] Linux
- [X] macOS
- [ ] Windows

### Obsidian Version

Version 1.6.7

### Plugin Version

0.5.0

### Checks

- [ ] I have tried it with all other plugins disabled and the error still occurs

**Comment by sims5487** [comment]
I'm also having this same issue, but just not with Instagram URLs - anything that is classified as "unreadable" doesn't seem to be saving. I first noticed the issue with this URL: https://obsidianttrpgtutorials.com/Obsidian+TTRPG+Tutorials/Getting+Started/Getting+Started

I'm also receiving the "@mozilla/readability considers this document to unlikely be readerable." message in Obsidian when pasting.

I also tried it with an Instagram URL and got the same result.

This isn't working for me on Mac, Windows, or iPhone. I'm also using OBsidian 1.6.7 and ReadItLater 0.5.0
